recipes/openmmtorchplugin/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set name = "openmmtorchplugin" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SCMusson/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 1a9197c32209eaa37b9dcb1a7ef6ed30841ea5beed124b902485c20f5dd92021
+
+build:
+  number: 0
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}   # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
+  skip: true  # [not linux or (linux and cuda_compiler_version in (undefined, 'None', '10.2'))]
+  rpaths:
+    - lib/
+    - {{ SP_DIR }}/torch/lib
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux64 or ppc64le]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - cmake
+    - make
+  
+  host:
+    - python
+    - pip
+    - swig
+    - openmm
+    - pytorch-gpu {{ pytorch_version }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - pytorch-cpu {{ pytorch_version }}  # [osx]
+    - ocl-icd  # [linux]
+    - khronos-opencl-icd-loader  # [osx]
+  run:
+    - python
+    - pytorch
+    - {{ pin_compatible('pytorch-gpu', max_pin='x.x.x') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - {{ pin_compatible('pytorch-cpu', max_pin='x.x.x') }}  # [osx]
```

```suggestion
    - {{ pin_compatible('pytorch-gpu', max_pin='x.x.x') }}  # [cuda_compiler_version not in (undefined, 'None')]
    - {{ pin_compatible('pytorch-cpu', max_pin='x.x.x') }}  # [osx]
```
Redundant package spec.